src/locale/locales/nl/messages.po

```diff
@@ -0,0 +1,7903 @@
+msgstr "Er is een fout opgetreden bij het selecteren van de video"
+
+#: src/screens/StarterPack/StarterPackScreen.tsx:347
+#: src/screens/StarterPack/StarterPackScreen.tsx:369
+msgid "An error occurred while trying to follow all"
+msgstr "Er is een fout opgetreden tijdens het proberen om allen te volgen"
+
+#: src/view/com/composer/state/video.ts:449
+msgid "An error occurred while uploading the video."
+msgstr "Er is een fout opgetreden bij het uploaden van de video."
+
+#: src/lib/moderation/useReportOptions.ts:28
+msgid "An issue not included in these options"
+msgstr "Een probleem niet vermeld in deze opties"
+
+#: src/components/hooks/useFollowMethods.ts:35
+#: src/components/hooks/useFollowMethods.ts:50
+#: src/components/ProfileCard.tsx:326
+#: src/components/ProfileCard.tsx:346
+#: src/view/com/profile/FollowButton.tsx:36
+#: src/view/com/profile/FollowButton.tsx:46
+msgid "An issue occurred, please try again."
+msgstr "Er is een probleem opgetreden. Probeer het opnieuw."
+
+#: src/components/dms/dialogs/NewChatDialog.tsx:36
+msgid "An issue occurred starting the chat"
+msgstr "Er is een probleem opgetreden bij het starten van de chat"
+
+#: src/components/dms/dialogs/ShareViaChatDialog.tsx:47
+msgid "An issue occurred while trying to open the chat"
+msgstr "Er is een probleem opgetreden bij het openen van de chat"
+
+#: src/screens/Onboarding/index.tsx:29
+#: src/screens/Onboarding/state.ts:81
+msgid "Animals"
+msgstr "Dieren"
+
+#: src/view/com/util/post-embeds/GifEmbed.tsx:149
+msgid "Animated GIF"
+msgstr "Geanimeerde GIF"
+
+#: src/lib/moderation/useReportOptions.ts:33
+msgid "Anti-Social Behavior"
+msgstr "Asociaal gedrag"
+
+#: src/view/screens/Search/Search.tsx:347
+#: src/view/screens/Search/Search.tsx:348
+msgid "Any language"
+msgstr "Elke taal"
+
+#: src/view/com/composer/threadgate/ThreadgateBtn.tsx:49
+msgid "Anybody can interact"
+msgstr "Iedereen kan reageren"
+
+#: src/view/screens/LanguageSettings.tsx:94
+msgid "App Language"
+msgstr "App-taal"
+
+#: src/view/com/modals/AddAppPasswords.tsx:138
+msgid "App Password names can only contain letters, numbers, spaces, dashes, and underscores."
+msgstr "App-wachtwoordnamen mogen alleen letters, cijfers, spaties, streepjes en onderstrepingstekens bevatten."
+
+#: src/view/com/modals/AddAppPasswords.tsx:103
+msgid "App Password names must be at least 4 characters long."
+msgstr "App-wachtwoordnamen moeten minimaal 4 tekens lang zijn."
+
+#: src/Navigation.tsx:285
+#: src/view/screens/AppPasswords.tsx:197
+#: src/view/screens/Settings/index.tsx:673
+msgid "App Passwords"
+msgstr "App-wachtwoorden"
+
+#: src/view/screens/AppPasswords.tsx:232
+msgid "App password deleted"
+msgstr "App-wachtwoord verwijderd"
+
+#: src/view/screens/Settings/index.tsx:664
+msgid "App password settings"
+msgstr "App-wachtwoordinstellingen"
+
+#: src/components/moderation/LabelsOnMeDialog.tsx:148
+#: src/components/moderation/LabelsOnMeDialog.tsx:151
+msgid "Appeal"
+msgstr "Beroep"
```

Comment:
"Bezwaar" to resemble user expectations migrating from other platforms. Unclear whether verb or noun. Leaving as is to see outcome in UI.

Reply:
In case it's helpful, this is a verb, as it's the text displayed in the button the user can click/tap if they want to appeal against a label that has been applied to their account or content.

E.g.:

![IMG_8967](https://github.com/user-attachments/assets/7f9f0941-cfc3-42d4-970a-cb9a348ddda4)

Reply:
We would not use a single word "Beroep" in that context, and we'd say "Maak bezwaar" or "In beroep gaan".

Reply:
@surfdude29 Thx for context. "appeal" -> "bezwaar maken" (to reduce length).